examples/test_steps_blending_rmi.py

```diff
@@ -0,0 +1,287 @@
+# -*- coding: utf-8 -*-
+
+from matplotlib import cm, pyplot as plt
+import numpy as np
+import xarray as xr
+import os
+from datetime import datetime
+from pprint import pprint
+from pysteps import io, rcparams
+from pysteps.utils import conversion, transformation, reprojection
+from pysteps.visualization import plot_precip_field
+
+# Set workdir and number of output time steps
+work_dir = "c:\\Users\\imhof_rn\\pysteps"
```

Comment:
remove absolute local path

Reply:
Oops, I see that this example script (that will later on, in a different version be used in the main_blending_module PR) ended up in this PR. I guess I was working on two PRs at the same time. I'll remove it, so that will directly solve the listed issues here. ;)